.travis.yml

```diff
@@ -1,3 +1,24 @@
 language: go
+sudo: false
+
+env:
+  - GO111MODULE=on
+
+git:
+  depth: 1
+
+go:
+  - 1.12.x
+  - 1.13.x
```

Comment:
Wasn't so sure what versions to support, I declared the latest version and one older one.